src/app/components/mno-app-install-btn/mno-app-install-btn.coffee

```diff
@@ -69,13 +69,17 @@ angular.module 'mnoEnterpriseAngular'
       vm.canProvisionApp = false
 
       vm.buttonDisabled = () ->
-        !vm.canProvisionApp || vm.appInstallationStatus() == "CONFLICT" || !vm.orderPossible
+        vm.billingDetailsRequired || !vm.canProvisionApp || vm.appInstallationStatus() == "CONFLICT" || !vm.orderPossible
 
       vm.updateButtonDisabledTooltip = () ->
-        if !vm.canProvisionApp
+        if vm.billingDetailsRequired
+          'mno_enterprise.templates.components.app_install_btn.billing_details_req'
+        else if !vm.canProvisionApp
           'mno_enterprise.templates.components.app_install_btn.insufficient_privilege'
         else if !vm.orderPossible
           'mno_enterprise.templates.dashboard.marketplace.show.no_pricing_plans_found_tooltip'
+        else if vm.conflictingApp
+          $translate.instant('mno_enterprise.templates.components.app_install_btn.conflicting_app') + ' ' + vm.conflictingApp.name
```

Comment:
Why is this one using a `$translate.instant`, could you just use interpolation in the locale string?